lib/util/affix.js

```diff
@@ -235,7 +235,7 @@ function affix(aff) {
 
       flags[ruleType] = value
     } else if (ruleType === keyType) {
-      flags[ruleType] = flags[ruleType].concat(parts[1].split('|'))
+      flags[ruleType].push.apply(flags[ruleType], parts[1].split('|'))
```

Comment:
Can `push.apply` be moved to the top, similar to `own = ...`?

As in:

```
var push = [].push
```

And then:

```suggestion
      push.apply(flags[ruleType], parts[1].split('|'))
```

Reply:
Sure. Actually I want to try optimizing that further, maybe always growing the larger array of the two would be beneficial, but I don't think nspell usually generates long-enough arrays for that to make a difference.

Reply:
Flags typically aren’t very long, but there are thousands of entries which include 1-5 of ’em.